doc/manpages/meson.build

```diff
@@ -0,0 +1,46 @@
+# Included from top-level meson.build
+
+# doc/manpages's Defines
+man5 = meson.current_build_dir() + 'dinit-service.5'
+man8 = [
+    meson.current_build_dir() + 'dinit.8',
+    meson.current_build_dir() + 'dinitcheck.8',
+    meson.current_build_dir() + 'dinitctl.8',
+    meson.current_build_dir() + 'dinit-monitor.8'
+]
+
+if install_shutdown_man
+    man8 += [
+        meson.current_build_dir() + 'shutdown.8',
+        meson.current_build_dir() + 'reboot.8',
+        meson.current_build_dir() + 'halt.8'
+    ]
+    find_program('ln')
```

Comment:
I don't think `find_program` does anything if you don't use its return. I'm pretty sure what you would need is something like:

```
ln_cmd = find_program('ln')
run_command(ln_cmd, 'shutdown.8', meson.current_build_dir() + 'reboot.8', check : true)
```
etc.

But, what is the need for `find_program` here anyway? It will just look on the PATH. If you say `run_command('ln', ...)` it will also look on the PATH.

Reply:
Seems like you right; I send a commit to fix many issues